File: C++/Fundamentals/Functions - Lab/Grades.cpp
#include <iostream>
using namespace std;
int main() {
	double grade;
	cin >> grade;
	if (grade >= 2.00 && grade <= 2.99) {
		cout << "Fail";
	}
	else if (grade >= 3.00 && grade <= 3.49) {
		cout << "Poor";
	}
	else if (grade >= 3.50 && grade <= 4.49) {
		cout << "Good";
	}
	else if (grade >= 4.50 && grade <= 5.49) {
		cout << "Very good";
	}
	else if (grade >= 5.50 && grade <= 6.00) {
		cout << "Excellent";
	}
	return 0;
}

File: Python/Fundamentals 2(2025-2026)(NEW)/Functions/Lab/Grades.py
# •	2.00 – 2.99 - "Fail"
# •	3.00 – 3.49 - "Poor"
# •	3.50 – 4.49 - "Good"
# •	4.50 – 5.49 - "Very Good"
# •	5.50 – 6.00 - "Excellent"


def grade(val = float(input())):

    if 2.00 <= val <= 2.99:
        print("Fail")

    elif 3.00 <= val <= 3.49:
        print("Poor")

    elif 3.50 <= val <= 4.49:
        print("Good")

    elif 4.50 <= val <= 5.49:
        print("Very Good")

    elif 5.50 <= val <= 6.00:
        print("Excellent")


grade()